doc/build-unix.md

```diff
@@ -61,13 +61,17 @@ Versions used in this release:
 
 Dependency Build Instructions: Ubuntu & Debian
 ----------------------------------------------
-sudo apt-get install build-essential
-sudo apt-get install libssl-dev
-sudo apt-get install libdb4.8-dev
-sudo apt-get install libdb4.8++-dev
- Boost 1.40+: sudo apt-get install libboost-all-dev
- or Boost 1.37: sudo apt-get install libboost1.37-dev
-sudo apt-get install libqrencode-dev
+- sudo apt-get install build-essential
+- sudo apt-get install libssl-dev
+- sudo apt-get install libdb4.8-dev
+- sudo apt-get install libdb4.8++-dev
+
+Boost 1.40+: 
+- sudo apt-get install libboost-all-dev
+
+Or Boost 1.37: 
+- sudo apt-get install libboost1.37-dev
+- sudo apt-get install libqrencode-dev
```

Comment:
Not trying to nitpick but if you are doing this let's move the libqrencode-dev option up under libdb4.8 instead of under boost.